rust/rsc/src/rsc/read_job.rs

```diff
@@ -59,7 +62,7 @@ pub async fn read_job(
         .transaction::<_, (Option<Uuid>, ReadJobResponse), DbErr>(|txn| {
             Box::pin(async move {
                 let Some(matching_job) = job::Entity::find()
-                    .filter(job::Column::Hash.eq(payload.hash()))
+                    .filter(job::Column::Hash.eq(hash))
                     .one(txn)
                     .await?
                 else {
```

Comment:
Can we add a `tracing::info!("miss")` here? or is that recorded somewhere else on the server side?